src/org/pentaho/reporting/platform/plugin/async/IAsyncReportListener.java

```diff
@@ -2,7 +2,7 @@
 
 import org.pentaho.reporting.engine.classic.core.event.ReportProgressListener;
 
-public interface IAsyncReportListener extends ReportProgressListener {
+public interface IAsyncReportListener extends ReportProgressListener, IAsyncReportState {
 
```

Comment:
Get rid of that. There is no need to have this interface implemented here, as no one who expects and IAsyncReportListener makes use of the IAsyncReportState functionality.